### ISSUE TITLE
TCD MS 1298 (H.2.7)

### ISSUE THREAD

**Issue by cyocum** [cyocum/irish-gen#185]
Add this to the database, according to @AdmiralHip M. E. Dobbs did a transcription and published it.  Update this ticket with that information once found.

**Comment by AdmiralHip** [comment]
She only did a transcription of a section, the reference is as follows: M.E. Dobbs, "Miscellany from H.2.7 (T.C.D.)," Zeitschrift fur celtische Philologie 21 (1938 - 1940): 307 - 318. A scan of the article can be found here: http://sulis.ucc.ie/cdi/textarchive-results2.php?keywords=M

**Comment by D-Groenewegen** [comment]
Hi Chris and Eystein, this is probably

Dobbs, Margaret E. (ed.), “Miscellany from H.2.7 (T.C.D.)”, *Zeitschrift
für celtische Philologie* 21 (1938–1940): 307–318.
which is available online from Celtic Digital Initiative
(https://www.vanhamel.nl/codecs/Dobbs_1938a2)

The transcription is not complete though. If I recall correctly, Donnchadh
Ó Corráin undertook a full transcription and intended it for CURIA but did
not get round to publishing it, although Bart Jaski was able to make use of
it in *Early Irish kingship and succession* and the dissertation that
formed the basis for this book.







Op wo 17 okt. 2018 om 16:41 schreef Christopher Yocum <
notifications@github.com>:

**Comment by D-Groenewegen** [comment]
Whoops, mid-air collision

Op wo 17 okt. 2018 om 17:16 schreef D G <dennisgroenewegen@gmail.com>:

**Comment by AdmiralHip** [comment]
I have a copy of the transcription Donnchadh did, but I am unsure as to the legality of using it. I received it from a mutual friend after he passed. I'm using it for my own PhD thesis at the moment though.

**Comment by D-Groenewegen** [comment]
Well then, I'd love to know more about your work one day!

Of course, simply publishing the unpublished material of a recently
deceased scholar, something you may have received in goodwill, is not
normally considered to be socially and academically acceptable, so I
wouldn't go as far as to suggest anything of the kind myself. Maybe those
in charge of his legacy already formulated plans for this. I do think that
it would be a fitting tribute to the man (esp. as a historian with
interests that include Irish genealogies and as the founding father of
CELT) if his work on this section could somehow be used for this project.

Anyway, before I say anything rash (if I haven't done so yet), this
probably calls for a more private discussion.


Op wo 17 okt. 2018 om 17:40 schreef AdmiralHip <notifications@github.com>:

**Comment by cyocum** [comment]
Yes, this has been discussed in private and we will **not** be using that particular transcription but using another one done specifically for this project with due attention paid to Donnchadh's.

**Comment by D-Groenewegen** [comment]
Glad to hear you worked things out in this way!

Op wo 17 okt. 2018 om 20:57 schreef Christopher Yocum <
notifications@github.com>:

**Comment by AdmiralHip** [comment]
Yes, so I'll be (slowly) working on getting a transcription up. The MS is quite long, but it has some really interesting stuff.
